foundry.toml

```diff
@@ -30,4 +30,10 @@ mainnet = "${ETHEREUM_RPC_URL}"
 goerli = "${GOERLI_RPC_URL}"
 polygon = "${POLYGON_RPC_URL}"
 
+
+[invariant]
+runs = 100
+depth =1
+fail_on_revert = true
```

Comment:
determines whether reverts during the fuzzing are acceptable.  usually, you probably want this set to false so that reverts ARE allowed... after all, given randomized inputs, reverts are expected and often the correct behavior of contracts.  However, certain tests could be designed in such a way and inputs can be bounded/assumed (forge cheatcodes) or manipulated such that reverts are NOT expected.  it just depends.  Handlers should usually be designed so that reverts aren't happening all of the time -- otherwise no actual state changes are being meaningfully tested.